src/crimson/os/seastore/random_block_manager/block_rb_manager.cc

```diff
@@ -143,6 +143,20 @@ BlockRBManager::write_ertr::future<> BlockRBManager::write(
     std::move(bp));
```

Comment:
Can you elaborate why this version requires a copy?

IIUC `CachedExtent::ptr` should be page aligned as well.

Reply:
Since this `bp` comes from `CachedExtent::ptr`, I thought that the `bp` could be freed when the extent is deallocated. Am I misunderstanding something?

Reply:
If it is the only consideration, take `write_ertr::future<> write(paddr_t addr, bufferptr pr)` instead of references should work for both cases without data copy, right?

Reply:
Right. Will do.